vue-components/public/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
This would be the dev entrypoint mentioned in the PR description. I think there is not too much point in having it (the `public` folder) in a library and we'll - as written - try and get storybook to a state where we can look at/develop the individual components.